charts/incubator/notifiarr/values.yaml

```diff
@@ -0,0 +1,44 @@
+image:
+  repository: tccr.io/truecharts/notifiarr
+  pullPolicy: IfNotPresent
+  tag: latest@sha256:913c71f5e325dd9af53c9e96f3d5d7d1b2d058a41f8253489320a4c69fbb0a44
+
+service:
+  main:
+    ports:
+      main:
+        port: 5454
+        protocol: http
+        targetPort: 5454
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            TMPDIR: /tmpdir
+            DN_API_KEY: "CHANGEME"
+            DN_UPSTREAMS_0: "172.16.0.0/16"
+            DN_UI_PASSWORD: "username:9CharacterPassword"
+
+persistence:
+  config:
+    enabled: true
+    mountPath: /config
+  utmp:
+    enabled: true
+    mountPath: /var/run/utmp
```

Comment:
I guess this was meant to be mapped to host to monitor something?
(If that's the case I'm not sure if scale allows mapping this host path)

Reply:
This one could probably be made optional, per docs: Map the /var/run/utmp volume if you want to count users.
https://notifiarr.wiki/Client/Installation

Reply:
The way it is defined now it will be a PVC

Reply:
I removed this, not sure it has enough value as a default config, people can add it if they want.